src/Model/Post.php

```diff
@@ -36,10 +36,10 @@ class Post
 	 *
 	 * @param integer $uri_id
 	 * @param array   $fields
-	 * @return int    ID of inserted post
+	 * @return bool   Success of the insert process
```

Comment:
Why the change?

Reply:
The `Post` table has got no auto increment field, so this always returned `0`. Also the other `Post` related `insert` commands (for the other tables) also only return a boolean.